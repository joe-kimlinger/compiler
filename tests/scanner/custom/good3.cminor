// characters
c: char = '\"'
c: char = '0';
c = '\0';
c = '\'';
