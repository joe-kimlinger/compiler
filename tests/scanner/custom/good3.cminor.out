COMMENT
IDENTIFIER c
COLON
CHAR
ASSIGN
CHARACTER_LITERAL "
IDENTIFIER c
COLON
CHAR
ASSIGN
CHARACTER_LITERAL 0
SEMICOLON
IDENTIFIER c
ASSIGN
CHARACTER_LITERAL 
SEMICOLON
IDENTIFIER c
ASSIGN
CHARACTER_LITERAL '
SEMICOLON
